clients/client-athssox/src/main/java/com/sitionix/bffssox/client/AuthClientImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.sitionix.bffssox.client;
+
+import com.app_afesox.bffssox.client.api.UserApi;
+import com.app_afesox.bffssox.client.dto.UserDTO;
+import com.app_afesox.bffssox.client.dto.UserResponseDTO;
+import com.sitionix.bffssox.domain.User;
+import com.sitionix.bffssox.mapper.UserClientMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class AuthClientImpl implements AuthClient {
```

Comment:
its not user friendly tto have such name, rename it

Reply:
Rename to UserClientImpl